src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -92,7 +102,46 @@ public DeltaBuilderTests()
 
             _cache = Substitute.For<IDeltaCache>();
 
+            Delta previousDelta = new Delta();
+            previousDelta.StateRoot = Keccak.EmptyTreeHash.Bytes;
+            _cache.TryGetOrAddConfirmedDelta(Arg.Any<Cid>(), out Arg.Any<Delta>()).Returns(x =>
+            {
+                x[1] = previousDelta;
+                return true;
+            });
+
             _dateTimeProvider = new DateTimeProvider();
+
+            IDb codeDb = new MemDb();
+            ISnapshotableDb stateDb = new StateDb();
+            ISpecProvider specProvider = new CatalystSpecProvider();
+            _cryptoContext = new FfiWrapper();
+            _stateProvider = new StateProvider(stateDb, codeDb, LimboLogs.Instance);
+            IStorageProvider storageProvider = new StorageProvider(stateDb, _stateProvider, LimboLogs.Instance);
+            KatVirtualMachine virtualMachine = new KatVirtualMachine(_stateProvider,
+                storageProvider,
+                new StateUpdateHashProvider(),
+                specProvider,
+                LimboLogs.Instance);
+            _deltaExecutor = new DeltaExecutor(specProvider,
+                _stateProvider,
+                storageProvider,
+                virtualMachine,
+                _cryptoContext,
+                _logger);
+        }
+
+        private readonly Dictionary<int, IPublicKey> _publicKeys = new Dictionary<int, IPublicKey>();
+
+        private IPublicKey GetPublicKey(int index)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GetPublicKey'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)